package.json

```diff
@@ -61,7 +63,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
```

Comment:
when is it going to run, do you have a sample of how it runs? a simple screenshot or screencast should be presented